backoffice/src/views/exports/certificates/LegacyCertificate.tsx

```diff
@@ -183,7 +183,9 @@ const MaritimeUse: FunctionComponent<UseProps> = ({
     <div className="use full">
       <h4 className="title use">
         {" "}
-        {`#${index} Beacon Use -  ${use.environment} `}
+        {`#${index} Beacon Use -  ${use.environment.toUpperCase()} ${
```

Comment:
Do we want the .toUpperCase to be a server-side operation so the front end can do less work; or are there places where we need use.environment to be sentence/lower case?